.github/ISSUE_TEMPLATE/1-bugreport.yml

```diff
@@ -0,0 +1,38 @@
+name: "Bug Report"
+description: "File a bug report"
+type: "Bug"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this bug report!
+  - type: textarea
+    id: version
+    attributes:
+      label: "Packages versions"
+      description: "Let us know the versions of any other packages used. For example, which version of the protocol are you using?"
+      placeholder: "miden-base: 0.1.0"
```

Comment:
nit: maybe we should change the description to just: "Let us know the versions of any other packages used." and change the placeholder to "Which version of the protocol are you using?".

Reply:
Agreed with the first. I would leave the placeholder as-is to demonstrate what we are after. Otherwise, users might e.g. say just `0.1` and we wouldn't know which package that refers to.